src/zope/traversing/api.py

```diff
@@ -58,32 +47,21 @@ def joinPath(path, *args):
 
 
 def getPath(obj):
-    """Returns a string representing the physical path to the object.
+    """
+    Returns a string representing the physical path to the object.
     """
     return ILocationInfo(obj).getPath()
 
 
 def getRoot(obj):
-    """Returns the root of the traversal for the given object.
+    """
+    Returns the root of the traversal for the given object.
     """
     return ILocationInfo(obj).getRoot()
 
 
 def traverse(object, path, default=_marker, request=None):
-    """Traverse 'path' relative to the given object.
-
-    'path' is a string with path segments separated by '/'.
-
-    'request' is passed in when traversing from presentation code. This
-    allows paths like @@foo to work.
-
-    Raises LocationError if path cannot be found
-
-    Note: calling traverse with a path argument taken from an untrusted
-          source, such as an HTTP request form variable, is a bad idea.
-          It could allow a maliciously constructed request to call
-          code unexpectedly.
-          Consider using traverseName instead.
+    """
     """
```

Comment:
~~Was this change intentional?  It doesn't look intentional.~~

This seemed peculiar before I saw the interface `__doc__` copying at the end of the module.

I still don't think having an explicit empty docstring was intentional.

Reply:
Yikes, no it wasn't, I was trying to leave at least the topic sentence. Good catch.